docs/extend/extend-desktop/mvd-server-config.md

```diff
@@ -193,29 +142,19 @@ To include Apps, be sure to define the location of the Plugins directory in the
 [See logger documentation](https://github.com/zowe/zlux/wiki/Logging#server-startup-logging-configuration)
 
 ## ZSS Configuration
-When running ZSS, it will require a JSON configuration file similar or the same as the one used for the App Server. The attributes that are needed for ZSS, at minimum, are: *productDir*, *siteDir*, *instanceDir*, *groupsDir*, *usersDir*, *pluginsDir* and **agent**. All of these attributes have the same meaning as described above for the App server, but if the App server and ZSS are not run from the same location, then these directories may be different if desired.
+When running ZSS, it will require a configuration file similar or the same as the one used for the App Server. The attributes that are needed for ZSS, at minimum, are: *productDir*, *siteDir*, *instanceDir*, *groupsDir*, *usersDir*, *pluginsDir* and **agent**. All of these attributes have the same meaning as described above for the App server, but if the App server and ZSS are not run from the same location, then these directories may be different if desired.
```

Comment:
I don't think is referring to internal server.json is it? And if it's referring to the Zowe configuration file, then we should make it clear theres only 1 Zowe configuration file, so maybe "It will require a component section with attributes similar or the same as the one used for the App Server" or
"It will require a configuration section with attributes similar or the same as the one used for the App Server"
 etc etc.

Reply:
its actually not true there will be only 1 zowe configuration file. they'll all share the same specification but it can be one monolithic file or a file per server or however which way you want to slice it since the collection of files merge into 1 at runtime. so, yes this is intentionally vague.